astronomer/providers/amazon/aws/sensors/emr.py

```diff
@@ -46,3 +49,48 @@ def execute_complete(self, context: Dict[Any, Any], event: Dict[str, str]) -> No
                 raise AirflowException(event["message"])
             self.log.info(event["message"])
         return None
+
+
+class EmrStepSensorAsync(EmrStepSensor):
+    """
+    Async (deferring) version of EmrStepSensor
+
+    Asks for the state of the step until it reaches any of the target states.
```

Comment:
what are the target states..? can you elaborate please

Reply:
Default target is completed. [L60](https://github.com/astronomer/astronomer-providers/pull/184/files#diff-8e76080095c5b7ce2c591d26af2bf9a5ce5ac966f6b1e6b16f513f09e2feb58eR60)

Reply:
Other values can be any state https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/emr.html#EMR.Client.describe_step